cypress/support/commands/pageTestFragment/metaData.js

```diff
@@ -0,0 +1,197 @@
+Cypress.Commands.add('retrieveMetaDataContent', (metaDataTag, content) => {
+  cy.get(metaDataTag).should('have.attr', 'content', content);
+});
+
+Cypress.Commands.add('facebookMeta', (fbAdmins, appID, articleAuthor) => {
+  cy.get('head meta[name="fb:admins"]').should(
+    'have.attr',
+    'content',
+    fbAdmins,
+  );
+  cy.get('head meta[name="fb:app_id"]').should('have.attr', 'content', appID);
+  cy.get('head meta[name="article:author"]').should(
+    'have.attr',
+    'content',
+    articleAuthor,
+  );
+});
+
+Cypress.Commands.add('metadataAssertion', () => {
+  cy.window().then(win => {
+    cy.get('head meta[name="description"]').should(
+      'have.attr',
+      'content',
+      win.SIMORGH_DATA.pageData.promo.summary ||
+        win.SIMORGH_DATA.pageData.promo.headlines.seoHeadline,
+    );
+    cy.get('head meta[name="og:title"]').should(
+      'have.attr',
+      'content',
+      win.SIMORGH_DATA.pageData.promo.headlines.seoHeadline,
+    );
+    cy.get('head meta[name="og:type"]').should(
+      'have.attr',
+      'content',
+      win.SIMORGH_DATA.pageData.metadata,
+    );
+    cy.get('head meta[name="article:published_time"]').should(
+      'have.attr',
+      'content',
+      new Date(win.SIMORGH_DATA.pageData.metadata.firstPublished).toISOString(),
+    );
+    cy.get('head meta[name="article:modified_time"]').should(
+      'have.attr',
+      'content',
+      new Date(win.SIMORGH_DATA.pageData.metadata.lastPublished).toISOString(),
+    );
+    cy.get('html').should(
+      'have.attr',
+      'lang',
+      win.SIMORGH_DATA.pageData.metadata.passport,
+    );
+  });
+});
+
+Cypress.Commands.add('metadataAssertionAMP', AMPURL => {
+  // This will only work if you visit the matching canonical
+  // url prior to running this.
+  // AMP overrides the Window data in window.SIMORGH_DATA. In order to get
+  // around this we visit the canonical page first to retrieve
+  // window.SIMORGH_DATA and use this to compare against the metadata
+  // served in the head of an AMP page.
+
+  cy.window().then(win => {
+    cy.visit(AMPURL);
+    cy.get('head meta[name="description"]').should(
```

Comment:
Same as above on the the head, I think they can be chained rather than so many cy.gets